configuration/components/tracing.libsonnet

```diff
@@ -91,8 +91,23 @@ function(params) {
         namespace: tracing.config.namespace,
         labels: newCommonLabels(component),
       },
-      spec: {
-        strategy: 'allinone',
+      spec: tracing.config.jaegerSpec,
+    } + {
+      spec+: {
+        ui+: {
+          options+: {
+            menu+: [
+              {
+                anchorTarget: '_self',
+                label: 'Log Out',
+                url: './login',
```

Comment:
This URL won't work unless https://github.com/observatorium/api/pull/274 is merged.  We might want to hold off on this one until we get a positive review on that one.